util/test/paratest.server

```diff
@@ -608,6 +608,9 @@ sub main {
             }
         } elsif (/^-valgrind/) {
             $valgrind = 1;
+            if (/^-valgrindexe/) {
+                $valgrind = 2;
+            }
```

Comment:
This is fine as is, but you could also have an
  "} elsif (/^-valgrindexe/) {"
before the
  "} elsif (/^-valgrind/) {"